habitat-baselines/habitat_baselines/config/experiments_hab3/pop_play_kinematic_oracle_humanoid_spot.yaml

```diff
@@ -30,6 +29,16 @@ defaults:
     - other_agent_gps
     - has_finished_oracle_nav
   - /habitat/task/actions@habitat.task.actions.agent_0_arm_action: arm_action
+  - /habitat/task/actions@habitat.task.actions.agent_0_rearrange_stop: rearrange_stop
```

Comment:
Can you do all changes to habitat-lab and habitat-baselines in a separate PR? And you should make them into `SIRo_dev`. That should merge first, and then we can rebase/cherry-pick to update `SIRo_hitl`.

Sorry, I know that is a pain. Basically, we want to treat the HITL codebase an almost like a separate repo. By having all our HITL commits as only touching `examples/siro_sandbox`, we are guaranteed trivial rebases in the future.

Reply:
Addressed here: https://github.com/facebookresearch/habitat-lab/pull/1510